syntax = "proto2";

package qcraft.planner;
option go_package = "qcraft-proto.ai/onboard/planner/scheduler/proto/smooth_reference_line";

message SmoothedReferenceLineResultProto {
  message LaneIdVec {
    repeated sfixed64 lane_id = 1;
  }
  repeated LaneIdVec lane_id_vec = 1;
}